apps/compat/templates/compat/index.html

```diff
@@ -19,7 +19,7 @@
   'all': _('All active add-ons'),
 } %}
 
-<form class="go" action="" id="compat-form">
```

Comment:
Just correcting, see this discussion: https://github.com/mozilla/olympia/pull/374#discussion_r20920917